### PR TITLE
Updating File Picker Schema

### PR DESCRIPTION
**Pull request by JCrew0**
This PR updates the File Picker config docs and updates the File Browser docs formatting.

### CODE REVIEW

File: docs/controls/file-pickers/v8-schema.md [hunk 1]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+    /**
+     * Configures the base type of item which will be displayed.
+     * Only items of this general type will be selectable.
+     */
+    mode?: "files" | "folders" | "all";
+    /**
+     * `filters` options: file extension, i.e. .xlsx, .docx, .ppt, etc.
+     * `filters` options: 'photo', 'folder', 'video', 'documentLibrary'
+     */
+    filters?: `.${string}`[];
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md]: This is wrong. Not everything begins with `.`

File: docs/controls/file-pickers/v8-schema.md [hunk 2]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+      oneDrive?: boolean;
+      /**
+       * Show "Recent".
+       */
+      recent?: boolean;
+      /**
+       * Show "Shared"
+       */
+      shared?: boolean;
+      /**
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:275]: Remove `teamsCloudStorage`

File: docs/controls/file-pickers/v8-schema.md [hunk 3]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
-    */
-   authentication: {};
-   /**
-    * Specifies what types of items may be picked and where they come from. Support for these features is inconsistent currently for external applications.
-    */
-   typesAndSources?: {
-      /**
-       * Specifies the general category of items picked. Switches between 'file' vs. 'folder' picker mode,
-       * or a general-purpose picker.
-       * @default 'all'
```
Comment (anarmawala) [inline docs/controls/file-pickers/v8-schema.md:116]: We probably want to keep this

File: docs/controls/file-pickers/v8-schema.md [hunk 4]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+       * Show "My organization".
+       * This pivot is only supported in OneDrive for Business
+       */
+      myOrganization?: boolean;
+
+      /**
+       * Show "Recent lists".
+       * This pivot is only supported in OneDrive for Business
+       */
+      recentLists?: boolean;
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md]: Remove `recentLists`

File: docs/controls/file-pickers/v8-schema.md [hunk 5]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+      /**
+       * Show "Bing search".
+       * This pivot is only supported in OneDrive for Business
+       */
+      bing?: boolean;
+      /**
+       * Show "My lists".
+       * This pivot is only supported in OneDrive for Business
+       */
+      myLists?: boolean;
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md]: Remove `myLists`

File: docs/controls/file-pickers/v8-schema.md [hunk 6]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+      teamsCloudStorage?: boolean;
+      /**
+       * Show "Quick access".
+       */
+      sharedLibraries?: boolean;
+      /**
+       * Show "Stock images".
+       * This pivot is only supported in OneDrive for Business
+       */
+      stockImages?: boolean;
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md]: Remove `stockImages`

File: docs/controls/file-pickers/v8-schema.md [hunk 7]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+      /**
+       * Show "Bing search".
+       * This pivot is only supported in OneDrive for Business
+       */
+      bing?: boolean;
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md]: Remove `bing`

File: docs/controls/file-pickers/v8-schema.md [hunk 8]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+   * Note that the default configuration accounts for the expected authentication capabilities of the host app.
+   * Depending on what else is enabled by the host, the host may be expected to provide tokens for more services and scopes.
+   */
+  typesAndSources?: {
+    /**
+     * Configures the base type of item which will be displayed.
+     * Only items of this general type will be selectable.
+     */
+    mode?: "files" | "folders" | "all";
+    /**
```
Comment (anarmawala) [inline docs/controls/file-pickers/v8-schema.md:225]: ``filter` should be `string[]`

We should keep the comment as well

File: docs/controls/file-pickers/v8-schema.md [hunk 9]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+      sharedWithMe?: {};
+      /**
+       * Indicates that File Picker should start in the user's photos.
+       * This pivot is only available in OneDrive for Consumer
+       */
+      photos?: {};
+      /**
+       * Indicates that File Picker should start in the user's lists.
+       */
+      myLists?: {};
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md]: Remove `myLists`

File: docs/controls/file-pickers/v8-schema.md [hunk 10]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+      /**
+       * Indicates that File Picker should start in the user's recent lists.
+       * This pivot is only supported in OneDrive for Business
+       */
+      recentLists?: {};
+    };
+    sortBy?: {
+      /**
+       * Name of the field *in SharePoint* on which to sort.
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:151]: Remove `recentLists`

File: docs/controls/file-pickers/v8-schema.md [hunk 11]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+     */
+    identifyParent?: boolean;
+    /**
+     * Whether or not the client app must wait for a 'configure' command to be sent by the host before rendering.
+     */
+    waitForConfiguration?: boolean;
+    /**
+     * Whether or not the client app can re-establish a connection to the host after a reload.
+     */
+    restartable?: boolean;
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md]: Remove `restartable`

File: docs/controls/file-pickers/v8-schema.md [hunk 12]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
       };
-      close?: {
-         /**
-         * A custom label to apply to the button to close the picker.
-         * The default is 'Cancel'.
-         * This string must be localized if provided.
+      /**
+       * Configures special handling for custom lists.
+       */
+      listItem?: {
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md]: Remove `listItem`

File: docs/controls/file-pickers/v8-schema.md [hunk 13]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+     * in the tray, nav, or command bar.
+     */
+    custom?: {
+      actions?: {
+        key: string;
+        /**
+         * Filters defining what types of items the action operates on.
+         * If specified, the action will only be available for items which match the given filters.
+         */
+        filters?: `.${string}`[];
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md]: Consider in-lining `ExtFilter` here.

File: docs/controls/file-pickers/v8-schema.md [hunk 14]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
+     */
+    pivots?: {
       /**
-       * Sets the default 'pick' behavior once the user selects items.
+       * Show "My files".
        */
-      pick?: {
-         /**
-         * A custom label to apply to the button to pick the items.
-         * The default varies based on `action`, but is typically 'Select'.
```
Comment (anarmawala) [inline docs/controls/file-pickers/v8-schema.md:161]: Add this back

File: docs/controls/file-pickers/v8-schema.md [hunk 15]
```diff
@@ -13,199 +13,650 @@ This outlines the full schema available to configure the picker. These options a
...
-         label?: string;
+        mode?: "host" | "none";
+        /**
+         * Selection criteria to which the item applies.
+         */
+        selection?: "single" | "multiple" | "current" | "none";
+      }[];
+    };
+  };
+  accessibility?: {
```
Comment (anarmawala) [inline docs/controls/file-pickers/v8-schema.md:452]: /**
    * Specifies accessibility cues such as auto-focus behaviors.
     */

**Review by ThomasMichon — APPROVED** [review]
Check for all `.${string}` occurrences and replace with `ExtFilter`.

File: docs/controls/file-pickers/v8-schema.md [hunk 16]
```diff
@@ -12,200 +12,611 @@ ms.localizationpriority: High
...
+    enabled?: boolean;
+    /**
+     * Indicates that the host app can handle 'claims' challenges.
+     */
+    claimsChallenge?: {
+      /**
+       * @default false
+       */
+      enabled?: boolean;
+    };
```
Comment (anarmawala) [inline docs/controls/file-pickers/v8-schema.md:211]: Add `tokens`

**Review by anarmawala — APPROVED** [review]
Left Comments!

File: docs/controls/file-pickers/v8-schema.md (hunk 14 above)
Comment (JCrew0) [reply docs/controls/file-pickers/v8-schema.md:161]: It's not taken out, see below. The comment is just re-worded in the SDK but still says the same thing

**Pull request merged**